modules/topicsFpdModule.js

```diff
@@ -232,17 +238,20 @@ export function hasGDPRConsent() {
 /**
  * function to load the iframes of the bidder to load the topics data
  */
-function loadTopicsForBidders() {
+export function loadTopicsForBidders() {
   HAS_GDPR_CONSENT = hasGDPRConsent();
+
   if (!HAS_GDPR_CONSENT || !HAS_DEVICE_ACCESS) {
     logInfo('Topics Module : Consent string is required to fetch the topics from third party domains.');
     return;
   }
+
   const topics = config.getConfig('userSync.topics') || bidderIframeList;
+
   if (topics) {
     listenMessagesFromTopicIframe();
     const randomBidders = getRandomBidders(topics.bidders || [], topics.maxTopicCaller || 1)
-    randomBidders && randomBidders.forEach(({ bidder, iframeURL }) => {
+    randomBidders && randomBidders.forEach(({ bidder, iframeURL, fetchUrl, fetchRate }) => {
       if (bidder && iframeURL) {
         let ifrm = document.createElement('iframe');
         ifrm.name = 'ifrm_'.concat(bidder);
```

Comment:
I am not a prebid expert but from Google's documentation, iframe is not longer required when retrieving a topic, (see https://developer.chrome.com/docs/privacy-sandbox/topics/demo/#the-topics-api-headers-demo).

And If it is still required in the prebid context (due to CORS?) would it be possible to:
- create iframes only for bidders that do require it
- send topics to all others

Reply:
yes, fetch or iframes could be chosen.  i think ssp's would all need their own iframe/fetch endpoint configurations for the current implementation of this pr.. 

for a scenario where topics are sent back to all others might be something that prebid as a community would have to decide on/figure out

Reply:
Hi Jason, I don't understand the need for SSPs to retrieve Topics via headers outside of the HTTP call for the bid-request.
If we focus only on this call, this would have minimal to no overhead on publisher pages.
Which use-case would not be covered here ?

Reply:
@github-xavier-bucchiotty the purpose of this module is to share the topics with all participants. You are making a non-germane feature request in your comment.

Reply:
Thanks for the feedback Patrick, sorry for the noise